source/kubernetes/direct-csi.rst

```diff
@@ -66,9 +92,84 @@ specify ``direct.csi.min.io`` as the
 
 .. important::
 
-   The Direct CSI driver assumes that all nodes in the Kubernetes cluster
+   The Direct CSI plugin assumes that all nodes in the Kubernetes cluster
    use consistent paths for locally-attached drives. You cannot specify arbitary
-   paths per node using the Direct CSI driver.
+   paths per node using the Direct CSI plugin.
+
+Use Direct CSI with MinIO Kubernetes Operator
+---------------------------------------------
+
+The :doc:`MinIO Kubernetes Operator </kubernetes/operator-reference>` uses the
+``spec.volumeClaimTemplate`` object to generate the required Persistant
+Volume Claims (``PVC``) for the MinIO tenant. To direct the operator
+to generate ``PVC`` using Direct CSI volumes, set
+``spec.zones.volumeClaimTemplate.storageClassName`` to ``direct.csi.min.io``.
+
+For a complete procedure on using the MinIO Kubernetes Operator to create a
+MinIO tenant that uses Direct CSI volumes, see :ref:`todo`.
+
+Use Direct CSI with MinIO Kubernetes Plugin
+-------------------------------------------
+
+The :doc:`MinIO Kubernetes Plugin </kubernetes/kubectl-plugin-reference>`
+(``kubectl-minio``) wraps the MinIO Kubernetes Operator (``operator``) and
+provides a simple interface through ``kubectl`` for deploying and managing MinIO
+resources. To use Direct CSI volumes when creating a MinIO tenant with the
+``kubectl-minio`` plugin, include the 
+:mc-cmd:`--storageclass <kubectl minio tenant create storageclass>` option as
+part of the :mc-cmd:`kubectl minio tenant create` command.
+
+For a complete procedure on using ``kubectl-minio`` to create a MinIO tenant
+that uses Direct CSI volumes, see :ref:`minio-kubectl-minio-create-tenant`.
+
+.. _minio-direct-csi-persistent-volume-generation:
+
+Persistent Volume Generation
```

Comment:
@wlan0 This section specifically covers PV generation, so I'd appreciate your eyeball in case I've lost the plot here.